.gitpod.Dockerfile

```diff
@@ -9,4 +9,4 @@ USER gitpod
 #
 # More information: https://www.gitpod.io/docs/config-docker/
 
-RUN nvm install 10 && nvm use --default 10 && npm install -g yarn firebase-tools @vue/cli
+# RUN nvm install 10 && nvm use --default 10 && npm install -g yarn firebase-tools @vue/cli
```

Comment:
this line was causing the gitpod to crash.

Reply:
could you please give me more details?

Reply:
This is a description of how to build a VM image. Gitpod allows you to build a custom image, which is pretty cool. However, due to the way the base image is built, the installation of nvm fails. The fix is to move it to the init script instead.